core/trino-main/src/main/java/io/trino/execution/CallTask.java

```diff
@@ -73,7 +74,14 @@ public String getName()
     }
 
     @Override
-    public ListenableFuture<?> execute(Call call, TransactionManager transactionManager, Metadata metadata, AccessControl accessControl, QueryStateMachine stateMachine, List<Expression> parameters)
+    public ListenableFuture<?> execute(
```

Comment:
Is this commit still required as a part of this PR if we're dropping the support for data definition tasks?

Reply:
It is needed for only one case: `CREATE TABLE a LIKE b` where b is redirected. See https://github.com/lxynov/trino/blob/7e70e245298d987cd867d739451f379cce45a670/core/trino-main/src/main/java/io/trino/execution/CreateTableTask.java#L164